hadoop-examples/build.gradle

```diff
@@ -44,7 +43,7 @@ dependencies {
 shadowJar {
     version = version
     dependencies {
-        include dependency("io.pravega:hadoop-connectors")
+        include dependency("io.pravega:pravega-connectors-hadoop")
```

Comment:
As the example uses classes from `pravega-client`, to make it work in my setting I have to add:
```
include dependency("io.pravega:pravega-client")
include dependency("io.pravega:pravega-common")
```

Reply:
Are you saying the hadoop connectors does not include shaded Pravega client libraries?

Reply:
This was a problem of my setting, so forget it. I tested the code of the PR and it works.